projects/canopy/src/lib/button/button.component.scss

```diff
@@ -40,10 +40,7 @@
 
   &__spinner {
     position: absolute;
-    left: 0;
-    right: 0;
-    top: 0;
-    bottom: 0;
+    inset: 0;
```

Comment:
Nice, I didn't know this property existed x) 
The only thing is that it's not supported by IE11 which I don't think should be an issue since we are not supporting it in our apps anymore, @andy-polhill have we decided if IE11 support is dropped for Canopy too?

Reply:
We don't support in our team either, I would suggest we drop it for Canopy if we haven't already

Reply:
good question, I think there may be a couple of outliers, but I'm happy to start moving away from IE11 too. We should document or raise an issue somewhere about IE11 support.